cob_default_robot_config/cob3-2/arm_joint_configurations.yaml

```diff
@@ -1,6 +1,6 @@
 joint_names: ["arm_1_joint","arm_2_joint","arm_3_joint","arm_4_joint","arm_5_joint","arm_6_joint","arm_7_joint"]
```

Comment:
The joints are still called `arm_x_joint` as well as the tf_frames are still called the same as defined in the corresponding yaml files. However, by setting the parameter `tf_prefix` in [robot.launch](https://github.com/ipa-srd-fg/cob_simulation/blob/indigo_dev/cob_gazebo/launch/robot.launch), it is possible to resolve the tf_frames in the source code e.g. by `tf_listener_.resolve(bla)` or by `tf::resolve(tf_prefix, bla)`, which is done in the [robot_state_publisher](https://github.com/ros/robot_state_publisher/blob/jade-devel/src/robot_state_publisher.cpp). I hope that answers your questions.